From: Christian Persch <chpe@src.gnome.org>
Date: Tue, 15 Jul 2025 21:37:36 +0200
Subject: prefs: Replace icon

The emblem icons were removed from adwaita, so use something else.

Origin: upstream, 3.56.3, commit:b73d031aba8c42a0d972f941d9b450d090ea484c
Bug: https://gitlab.gnome.org/GNOME/gnome-terminal/-/issues/8126
Bug-Debian: https://bugs.debian.org/1109086
---
 src/terminal-prefs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/terminal-prefs.cc b/src/terminal-prefs.cc
index 437c58a..a100e46 100644
--- a/src/terminal-prefs.cc
+++ b/src/terminal-prefs.cc
@@ -487,7 +487,7 @@ listbox_create_row (const char *name,
 
   GtkStack *home_stack = GTK_STACK (gtk_stack_new ());
   gtk_widget_set_margin_start (GTK_WIDGET (home_stack), 12);
-  GtkImage *home_image = GTK_IMAGE (gtk_image_new_from_icon_name ("emblem-default-symbolic", GTK_ICON_SIZE_BUTTON));
+  GtkImage *home_image = GTK_IMAGE (gtk_image_new_from_icon_name ("object-select-symbolic", GTK_ICON_SIZE_BUTTON));
   gtk_widget_set_tooltip_text (GTK_WIDGET (home_image), _("This is the default profile"));
   gtk_stack_add_named (home_stack, GTK_WIDGET (home_image), "home");
   GtkLabel *home_label = GTK_LABEL (gtk_label_new (""));
